caleb-lab/lib/body-parser.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

A top-level comment to let you know you're missing some of these bullet points:

- add a package.json
- add a eslintrc
- add a gitignore
- add a readme with project description
- include any npm scripts for starting server, linting, testing, etc
- test your code
- ensure that all of your methods have test coverage
- write tests which start your server, send and receive, and confirm functionality